spec/controllers/commodities_controller_spec.rb

```diff
@@ -65,8 +64,7 @@
         end
 
         it 'redirects to actual version of the commodity page' do
-          expect(response.status).to eq 302
-          expect(response.location).to eq commodity_url(id: commodity_id.first(10))
+          expect(response).to redirect_to commodity_url(id: commodity_id.first(10))
```

Comment:
`expect(response.status).to eq 302` seemed a genuine test, should we keep it?

Reply:
My conclusion was we're probably not fussed about the particular status code, just that its in the 3xx range

That matcher chains through to the rails one, which checks a `3xx` status code, and checks the `response.location` ?

https://github.com/rails/rails/blob/83217025a171593547d1268651b446d3533e2019/actionpack/lib/action_dispatch/testing/assertions/response.rb#L53

Reply:
I got your point, yes it sounds good to me! 👍